### DISCUSSION TITLE
Why ignore flag is Deprecated

### DISCUSSION THREAD

**Discussion by dharmjit** [kisielk/errcheck#199] [General]
Hello, 

I was going through the Readme and couldn't understand why the ignore flag is deprecated https://github.com/kisielk/errcheck#the-deprecated-method.

**Comment by kisielk** [comment]
`-exclude` is the preferred way to achieve this now. `-ignore` just updates the list created with `-exclude` with a preset regexp

**Reply by dharmjit** [reply]
It requires maintaining a file which seems little addition in a monorepo.

**Reply by kisielk** [reply]
Yes, but it allows much finer-grained and more expressive control. For now we're keeping the `-ignore` flag so you can use it if you like, but it's not the preferred way.
